### PR TITLE
Fix/doc 4273

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Manage-Space.rst [hunk 1]
```diff
@@ -227,31 +227,37 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all informations related to the space creation.
 
 |image23|
 
+3. The space creator have to enter a space name (required and from 3 to 200 characters)
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
3. The space creator must enter a space name (3 to 200 characters) 
```

File: docs/Manage-Space.rst [hunk 2]
```diff
@@ -227,31 +227,37 @@ follows:
...
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all informations related to the space creation.
 
 |image23|
 
+3. The space creator have to enter a space name (required and from 3 to 200 characters) 
+   and description (from 0 to 255 characters).
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   and may enter an optional description (max 255 characters).
```

File: docs/Manage-Space.rst [hunk 3]
```diff
@@ -227,31 +227,37 @@ follows:
...
+
 .. note:: Special characters are not allowed on space names. Only the character **&** is allowed.
            
            |image75|
 
-.. _access-level-step:
+.. _space-templates:
 
-3. In the Access & Edit tab, select an access level.
+4. Space templates pulldown will presents all predefined templates that the space creator can use.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
4. A pulldown lists all predefined templates that the space creator can use. 
```

File: docs/Manage-Space.rst [hunk 4]
```diff
@@ -227,31 +227,37 @@ follows:
...
 .. note:: Special characters are not allowed on space names. Only the character **&** is allowed.
            
            |image75|
 
-.. _access-level-step:
+.. _space-templates:
 
-3. In the Access & Edit tab, select an access level.
+4. Space templates pulldown will presents all predefined templates that the space creator can use. 
+A description of every template in the pulldown is ensured and displayed just below the templates selector so that the space creator can differentiate between all templates.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
A description of each template is displayed below the selectors.
```

File: docs/Manage-Space.rst [hunk 5]
```diff
@@ -227,31 +227,37 @@ follows:
...
+
+.. note:: Every template have its specific visibility, regitration criteria and applications.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
.. note:: Each template defines its default visibility, registration criteria and list of applications.
```

File: docs/Manage-Space.rst [hunk 6]
```diff
@@ -227,31 +227,37 @@ follows:
...
+
+.. _access-level-step:
 
-|image31|
+4. The space creator can modify the predefined visibility and registration of it space.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
4. The space creator may override the predefined visibility and registration options of the space.
```

File: docs/Manage-Space.rst [hunk 7]
```diff
@@ -227,31 +227,37 @@ follows:
...
 
 .. _About-space-visibility:
 
--  **Visibility**: By default, the Visibility value is set to be Visible.
+-  **Visibility**: By default, the Visibility value is set to be the one related to the template.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
-  **Visibility**: By default, the Visibility value is set to be the one predefined in the template.
```

File: docs/Manage-Space.rst [hunk 8]
```diff
@@ -227,31 +227,37 @@ follows:
...
 
 	-  Visible: The space is always visible in the public spaces list.
 
 	-  Hidden: The space is not visible in the public spaces list.
 
--  **Registration**: By default, the Registration value is set as
-   Validation.
+-  **Registration**: By default, the Registration value is set to the one related to the template.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
-  **Registration**: By default, the Registration value is set to the one predefined in the template.
```

File: docs/Manage-Space.rst [hunk 9]
```diff
@@ -261,41 +267,86 @@ follows:
    -  Close: The user cannot request for joining, but only the space
       manager can invite him.
 
-4. In the **Invite users from group** tab, select all members of a specific
-group for your space where you are already 'manager'.
+5. In the **Users ** field, the space creator can add users or space members to the space to create just by adding them in the selector.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```

File: docs/Manage-Space.rst [hunk 10]
```diff
@@ -261,41 +267,86 @@ follows:
...
 
-One of the two following cases occurs when you select this tab.
+All users to add in the space are displayed by a suggestor to facilitate their add.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
Type-ahead suggestions allow to facilitate selection users and groups to invite.
```

File: docs/Manage-Space.rst [hunk 11]
```diff
@@ -261,41 +267,86 @@ follows:
...
 
--  **The first instance**: You have been already the manager of a group. It means that you have created at least a space.
+.. note:: The space creator can display only spaces that he is a member on them.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
.. note:: Only spaces that the creator is member of can be selected.
```

File: docs/Manage-Space.rst [hunk 12]
```diff
@@ -261,41 +267,86 @@ follows:
...
+.. _Space-templates:
 
-      You will see your selected group as below.
+**Space Templates**
 
-      |image25|
+Space templates concept is not directly exposed to users. From their perspective, they simply pick a type when creating a space.
 
-.. note:: To remove the selected groups, untick the **Select a group of people to invite in your space** checkbox.
+If the space creator choose a template from the templates pulldown, a description of the selected template will be displayed below the pulldown:
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
If the space creator choose a template from the pulldown, a description of the selected template is displayed below the pulldown:
```

File: docs/Manage-Space.rst [hunk 13]
```diff
@@ -261,41 +267,86 @@ follows:
...
+   -  Community: A general purpose area for collaboration and communication of a digital community.
+   -  Projects: A project space where members coordinate on tasks toward a predefined outcome.
+   -  Communication : Interpersonal communication where a space groups are involved in exchange of ideas, skills and interests.
+   -  Team: A central destination for members of a team.
 
-   After you have selected one existing group, all users in that group will be invited to join your newly created space.
+   .. note:: By default, in the create space popup we will have the Community template as default value.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
   .. note:: The default template for new spaces is:  Community

File: docs/Manage-Space.rst [hunk 14]
```diff
@@ -344,6 +395,12 @@ One of the two following cases occurs when you select this tab.
           Show More bar will appear at the page bottom. Click Show More
           to see more members.
 
+.. note:: In the space seeting, you can have an idea about the space template
```
Comment (plamarque) [inline docs/Manage-Space.rst:416]:
```suggestion
.. note:: In the space settings, you can get an idea of the original space template used to create
```

File: docs/Manage-Space.rst [hunk 15]
```diff
@@ -344,6 +395,12 @@ One of the two following cases occurs when you select this tab.
           Show More bar will appear at the page bottom. Click Show More
           to see more members.
 
+.. note:: In the space seeting, you can have an idea about the space template 
+          for your space, the Space template pulldown will display the selected template and
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
         your space, by looking at the Template field 
```

File: docs/Manage-Space.rst [hunk 16]
```diff
@@ -344,6 +395,12 @@ One of the two following cases occurs when you select this tab.
           Show More bar will appear at the page bottom. Click Show More
           to see more members.
 
+.. note:: In the space seeting, you can have an idea about the space template 
+          for your space, the Space template pulldown will display the selected template and 
+		  the pulldown will be disabled.
```
Comment (plamarque) [inline docs/Manage-Space.rst]: but you can't change it.

**Review by plamarque — DISMISSED** [review]
english language review

**Review by plamarque — COMMENTED** [review]
in docs/images/social/add_space_settings_tab.png please crop around the popup

File: docs/Manage-Space.rst [hunk 17]
```diff
@@ -344,6 +395,11 @@ One of the two following cases occurs when you select this tab.
           Show More bar will appear at the page bottom. Click Show More
           to see more members.
 
+.. note:: In the space settings, you can get an idea of the original space template used to create
+         your space, by looking at the Template field 
+		  the pulldown will be disabled.
```
Comment (plamarque) [inline docs/Manage-Space.rst]:
```suggestion
		  but you can't change it.
```

File: docs/Manage-Space.rst [hunk 18]
```diff
@@ -227,31 +227,37 @@ follows:
 
 1. Click |image22| at the top left corner of the **Space** page to open the Add New Space form.
 
-2. In the Settings tab, enter a space name (required and from 3 to 200 characters) 
-   and description (from 0 to 255 characters).
+2. A popup is displayed containing all informations related to the space creation.
```
Comment (mzorai) [inline docs/Manage-Space.rst]: The word information is uncountable, it does not take "s".

File: docs/Manage-Space.rst [hunk 19]
```diff
@@ -227,31 +227,37 @@ follows:
...
 
 |image23|
 
+3. The space creator must enter a space name (3 to 200 characters) 
+   and may enter an optional description (max 255 characters).
```
Comment (mzorai) [inline docs/Manage-Space.rst:235]: I think that we should precise the minimum characters which is 0 for description i.e. it not mandatory to fill it.

File: docs/Manage-Space.rst [hunk 20]
```diff
@@ -227,31 +227,37 @@ follows:
...
-3. In the Access & Edit tab, select an access level.
+4. A pulldown lists all predefined templates that the space creator can use. 
+A description of each template is displayed below the selectors.
 
-|image31|
+.. note:: Each template defines its default visibility, registration criteria and list of applications.
+
+.. _access-level-step:
+
+4. The space creator may override the predefined visibility and registration options of the space.
```
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
5. The space creator may override the predefined visibility and registration options of the space.
```

File: docs/Manage-Space.rst [hunk 21]
```diff
@@ -261,41 +267,86 @@ follows:
    -  Close: The user cannot request for joining, but only the space
       manager can invite him.
 
-4. In the **Invite users from group** tab, select all members of a specific
-group for your space where you are already 'manager'.
+5. In the **Users ** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
6. In the **Users** field, the space creator can invite users or space members to the space to create just by picking them with the selector.
```

File: docs/Manage-Space.rst [hunk 22]
```diff
@@ -261,41 +267,86 @@ follows:
...
 
-One of the two following cases occurs when you select this tab.
+Type-ahead suggestions allow to facilitate selection users and groups to invite.
 
--  **The first instance**: You have been already the manager of a group. It means that you have created at least a space.
+.. note:: Only spaces that the creator is member of can be selected.
 
-   |image24|
+6. Click Create Space button to finish adding your new space. The new space appears.
```
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
7. Click Create Space button to finish adding your new space. The new space appears.
```

File: docs/Manage-Space.rst [hunk 23]
```diff
@@ -261,41 +267,86 @@ follows:
...
+.. _Space-templates:
 
-      You will see your selected group as below.
+**Space Templates**
 
-      |image25|
+Space templates concept is not directly exposed to users. From their perspective, they simply pick a type when creating a space.
 
-.. note:: To remove the selected groups, untick the **Select a group of people to invite in your space** checkbox.
+If the space creator choose a template from the pulldown, a description of the selected template is displayed below the pulldown:
```
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
If the space creator chooses a template from the pulldown, a description of the selected template is displayed below the pulldown:
```

**Review by mzorai — CHANGES_REQUESTED** [review]

File: docs/Manage-Space.rst (hunk 19 above)
Comment (hghorbel) [reply docs/Manage-Space.rst:235]: While we have "optional" description, it means that the minimum is 0.

File: docs/Manage-Space.rst (hunk 18 above)
Comment (mzorai) [inline docs/Manage-Space.rst]:
```suggestion
2. A popup is displayed containing all information related to the space creation.
```

**Review by mzorai — CHANGES_REQUESTED** [review]

**Review by mzorai — APPROVED** [review]

**Pull request merged**